docs/source/models/supported_models.rst

```diff
@@ -442,6 +442,12 @@ Text Generation
     - :code:`Qwen/Qwen-VL`, :code:`Qwen/Qwen-VL-Chat`, etc.
     -
     - ✅︎
+  * - :code:`Qwen2AudioForConditionalGeneration`
+    - Qwen2-Audio
+    - T + A\ :sup:`E+`
```

Comment:
Do you intend to support embedding inputs? It doesn't look like you're handling this in the input mapper/processor.

Reply:
No, we haven't supported yet. I've removed `E` in the docs.